src/tasks.ts

```diff
@@ -1,18 +1,16 @@
 import { right } from "fp-ts/lib/Either";
 import { Task } from "fp-ts/lib/Task";
-import { fromLeft, TaskEither } from "fp-ts/lib/TaskEither";
+import * as T from "fp-ts/lib/Task";
+import { TaskEither } from "fp-ts/lib/TaskEither";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
 import { Millisecond } from "./units";
 
 /**
  * Returns a Task that resolves to a value after a delay.
  */
 export const delayTask = <A>(n: Millisecond, a: A): Task<A> =>
-  new Task<A>(
-    () =>
-      new Promise<A>(resolve => {
-        setTimeout(() => resolve(a), n);
-      })
-  );
+  T.delay(n)(T.of(a));
```

Comment:
nice ;)

Reply:
https://github.com/gcanti/fp-ts/blob/master/src/Task.ts#L85

Reply:
Since we now have a native delay method I'm wondering if we can just get rid of this wrapper.

Reply:
@gunzip do you mean [`timersPromises.setTimeout`](https://nodejs.org/docs/latest-v16.x/api/timers.html#timers_timerspromises_settimeout_delay_value_options)? They are available from Node 16